gwpy/frequencyseries/frequencyseries.py

```diff
@@ -269,6 +270,38 @@ def zpk(self, zeros, poles, gain, analog=True):
         """
         return self.filter(zeros, poles, gain, analog=analog)
 
+    def interpolate(self, df, left=0, right=0):
+        """Interpolate this `FrequencySeries` to a new resolution.
+
+        Parameters
+        ----------
+        df : `float`
+            desired frequency resolution of the interpolated `FrequencySeries`,
+            in Hz
+
+        left : `float`
+            value to return for f < flow, default: 0
+
+        right : `float`
+            value to return for f > fhigh, default: 0
+
+        Returns
+        -------
+        out : `FrequencySeries`
+            the interpolated version of the input `FrequencySeries`
+
+        See Also
+        --------
+        :func:`numpy.interp` for the underlying 1-D linear interpolation scheme
```

Comment:
This usage of `see also` doesn't work, numpydoc expects the format to be:

```
See Also
--------
:func:`numpy.interp`
    for the undelying 1-D linear interpolation scheme
```
I think. The new [numpydoc docs](https://numpydoc.readthedocs.io/en/latest/format.html#sections) suggest that you might be able to use a simple colon-separated key, value listing, but I haven't tested that.

Reply:
Good catch! This is addressed in commit 58c95a9.